README.md

```diff
@@ -1,114 +1,80 @@
 [![Open in GitHub Codespaces](
   https://img.shields.io/badge/Open%20in%20GitHub%20Codespaces-333?logo=github)](
   https://codespaces.new/dwave-examples/employee-scheduling?quickstart=1)
-[![Linux/Mac/Windows build status](
+<!-- [![Linux/Mac/Windows build status](
   https://circleci.com/gh/dwave-examples/employee-scheduling.svg?style=shield)](
-  https://circleci.com/gh/dwave-examples/employee-scheduling)
+  https://circleci.com/gh/dwave-examples/employee-scheduling) -->
 
-# Building an Employee Schedule
+# Employee Scheduling Demo
 
-Building a schedule for employees can be an extremely complex optimization
-problem in which managers must balance employee preferences against schedule
-requirements. In this example, we show how a discrete quadratic model (DQM) 
-and a constrained quadratic model (CQM) can be used to model this problem and
-how the hybrid solvers available in Leap can optimize over these competing 
-scheduling and preference needs.
+Employee scheduling is a common industry problem that can often become complex
+due to real-world constraints and requirements. In this example, we demonstrate
+a scheduling scenario with a variety of employees and rules.
 
-## Usage
+## Running the Demo
 
-To run the CQM demo, type the command:
+To run the demo, open the terminal and run `python app.py`. Open a web browser
+and copy the URL from the terminal to load the demo in the browser.
 
-```python demo.py```
+Once we have set all of the options to our satisfaction, click the "Solve CQM"
+button on the left. In the terminal we see status updates for "Building CQM..."
+and "Submitting CQM...".
 
-A prompt will appear asking for the number of employees:
+Once the problem has completed, the best solution returned will be displayed in
+place of "Employee Availability". Click back to the original availability using
+the tabs at the top of the schedule card.
```

Comment:
```suggestion
### Inputs
Input options under the Basic tab are as follows:

- Number of employees: Schedules always include 2 managers  and 1 trainee.
- Random seed: Optional; use if you want consistency between subsequent runs of the example.
- Example scenario: Auto-populates all settings with scenarios of varying
  sizes that produce feasible solutions.

Additional input options under the More Options tab are as follows:

- Allow isolated days off: Unchecked, this option means that employees are
  scheduled at least two consecutive days off between work days. 
- Require a manager on every shift: Checked, this option means that every shift
  must have exactly one manager on duty to supervise.
- Min/max shifts per employee: The range you set determines the number of shifts an
  employee can work in the month.
- Min/max employees per shift: The range you set determines how many employees need
  to be assigned to each shift.
- Max consecutive shifts: The maximum number of consecutive shifts an employee
  can be scheduled before a day off must be scheduled.

### Outputs

Once the problem has completed, the best solution returned is displayed in
place of "Employee Availability". Click back to "Employee Availability" using
the tabs at the top of the schedule card.
```

Recently Murray updated our [example template](https://github.com/dwave-examples/template) and Catherine has begun updating examples to fit the template, so it's worth adhering to the template with new/refreshed examples

Note that you can edit this suggested update (before or after accepting) if you do not accept the earlier suggestions on design using tabs instead of the collapsable button or want different text.